### ISSUE TITLE
separate P_Grid accessory data into positive and negative

### ISSUE THREAD

**Issue by wayn3** [Stog/homebridge-fronius-inverter#1]
The API object 'P_Grid' is a signed integer however when adding to Homekit it is displayed as an unsigned integer, it would be nice to be able to split positive and negative values out into 2 discrete accessories 
-P_Grid = Live Grid Export
+P_Grid = Live Grid Import

From here it would be awesome to have a daily tally of grid imported and grid exported values
I haven't investigated if this would be possible to get these import/export tallies with a direct API call but I'll look into it

**Comment by Stog** [comment]
Yes I am part way through building in functionality to provide a live grid consumption or feed in to grid, this should be ready soon. However from the API documentation I don't see any easy way of getting a daily import/export total.

**Comment by wayn3** [comment]
Had a bit of a look and I also don't see any easy way to pull the data from the API, might it be possible to log the data using [fakegato-history](https://www.npmjs.com/package/fakegato-history).

As well as this it might be worth having an additional sensor trigger when reaching a certain amount of grid feed is achieved, ie. maxing out. 

This way we could trigger actions based on energy now being wasted discarded.
